src/services/mondomarket/index.jsx

```
@@ -391,93 +388,94 @@ export const MondoDetailsViewer = ({
-              <MdBookmark size={30} className={`${textColor}`} />
-              <div>Book Mark</div>
+                <p className="text-gray-700 dark:text-dark_text_1 mt-4">
+                  {mondo?.description}
+                </p>
+              </div>
+
+              <div className="flex justify-center cursor-pointer gap-5 items-center mt-8">
+                <div
+                  className={`offer-button hover:${bgColor} hover:text-white text-light_text_1 dark:text-dark_text_1`}
+                  onClick={() => setOfferModal(true)}
+                >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)